tools/pfe-tools/test/config.ts

```diff
@@ -1,12 +1,11 @@
-import type { PfeDevServerConfigOptions } from './dev-server';
 import type { TestRunnerConfig } from '@web/test-runner';
 
 import { playwrightLauncher } from '@web/test-runner-playwright';
 import { summaryReporter, defaultReporter } from '@web/test-runner';
 import { junitReporter } from '@web/test-runner-junit-reporter';
 import { a11ySnapshotPlugin } from '@web/test-runner-commands/plugins';
 
-import { pfeDevServerConfig } from './dev-server.js';
+import { pfeDevServerConfig, type PfeDevServerConfigOptions } from '../dev-server/config.js';
```

Comment:
Another TIL:  I was unaware you could add a type to a regular import line, very useful for keeping things concise.